src_c/base.c

```diff
@@ -2094,11 +2095,21 @@ static PyMethodDef _base_methods[] = {
      "return an array struct interface as an interface dictionary"},
     {NULL, NULL, 0, NULL}};
 
+#if defined(BUILD_STATIC) && defined(NO_PYGAME_C_API)
+// in case of wasm+dynamic loading it could be a trampoline in the globals
+// generated at runtime.
+// when building static make global accessible symbol directly.
+static PyObject *pgExc_SDLError;
```

Comment:
I think it's fine if this pointer would be unconditionally global.. would make the code simpler+any bugs related to this could be caught even on regular cpython

Reply:
i agree, but i'm not sure of the shields to put there because when C-API is active the global is a MACRO pointing to the slot.
unconditionally global would conflict with the _pygame.h macro https://github.com/pygame/pygame/blob/d72f4be879e47f426b565f56a5b5b89edc8aa12b/src_c/include/_pygame.h#L95

Reply:
Well I think there are some header guards that prevent this confict, IIRC